cmd/hactar/commands/commands.go

```diff
@@ -0,0 +1,81 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/hactar"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/lotus/services"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/stats"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/token"
+	"github.com/mkideal/cli"
+	log "github.com/sirupsen/logrus"
+)
+
+// root command
+type rootT struct {
+	cli.Helper
+}
+
+var RootCommand = &cli.Command{
+	Argv: func() interface{} { return new(rootT) },
+	Desc: "Command line interface for Filecoin nodes monitoring deamon app",
+	Fn: func(ctx *cli.Context) error {
+		return nil
+	},
+}
+
+// start command
+type startT struct {
+	cli.Helper
+	Username string `cli:"u,username" usage:"lotus account" prompt:"Enter Lotus account"`
+	Password string `pw:"p,password" usage:"password of lotus account" prompt:"Enter Lotus password"`
+}
+
+var StartCommand = &cli.Command{
+	Name: "start",
+	Desc: "Start hactar daemon application",
+	Text: "",
+	Fn: func(ctx *cli.Context) error {
+		argv := ctx.Argv().(*startT)
+		// authenticate
+		if err := hactar.Auth(argv.Username, argv.Password); err != nil {
+			log.Error("Failed to authenticate to Lotus service.")
+		}
+		log.Info("Successful authentication.")
+		// detect miners and allow user to choose actor address
+		lotusService := services.NewLotusService(nil, nil)
+		actorAddress, err := lotusService.GetMinerAddress()
+		if err != nil {
+			fmt.Print("Worker down!")
+			return nil
+		}
+		log.Info("Actor address: ", actorAddress)
+		// display token and URL
+		token.DisplayToken()
+		// save node to backend
+		hactar.SaveNode(hactar.Node{
+			Token:        token.ReadTokenFromFile(),
+			Url:          "temp-url-holder",
```

Comment:
This is supposed to be IP address. Can you read it?

Reply:
Yes, will change this.